life_server/Functions/Actions/fn_pickupAction.sqf

```diff
@@ -2,23 +2,21 @@
 /*
     File: fn_pickupAction.sqf
     Author: Bryan "Tonic" Boardwine
-
     Description:
     Validates that the cash is not a lie
 */
-params [
-    ["_obj",objNull,[objNull]],
-    ["_client",objNull,[objNull]],
-    ["_cash",false,[true]]
-];
+private _target = param [0, objNull, [objNull]];
+
+if (!isRemoteExecuted) exitWith {};
+if (isNull _obj) exitWith {};
+if (!(_target getVariable ["item", false]) isEqualType []) exitWith {};
```

Comment:
you get the same var twice -> big nono

```suggestion
private _item = _target getVariable "item";
if (isNil "_item") exitWith {};
```

-> change line 17 to `_item` instead of getvar again